samples/MvcSample.Web/Startup.cs

```diff
@@ -103,7 +103,7 @@ public void Configure(IApplicationBuilder app)
             }
 #endif
 
-            app.UseInMemorySession();
+            app.UseSession();
```

Comment:
`UseInMemorySession` extension is going away and also the default is in-memory session.